README.md

```diff
@@ -1,56 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# JavaScript-API-based-webapp
 
-> Description the project.
+> This is a web application based on an external API. The external API provides data about TV shows. 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS & JavaScript
+- Webpack & Jest
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://myaserkhan.github.io/JavaScript-API-based-webapp/dist)
 
+## Project Documentation 📄
 
-## Getting Started
+[Presentation video link](https://drive.google.com/file/d/1D8PLAAdviYUUBaUcYYhe-b3VzwJ8Jajl/view?usp=sharing)
```

Comment:
- [ ] @myaserkhan and @nunegabriel great work on your presentation, however, your video is longer than the project requirement and you forgot to describe the objective of the project. Kindly record another video that is 5 min or less and ensures all the requirements are met as outlined in the project requirement as shown below:
![image](https://user-images.githubusercontent.com/61408860/198311078-488fe182-8108-4620-8883-ce787634ea01.png)